drone-ci/drone/resources.tf

```diff
@@ -121,14 +72,9 @@ resource "aws_volume_attachment" "default" {
   force_detach = true
 
   provisioner "local-exec" {
-    command = "echo \"[drone]\n${aws_instance.drone.public_ip}\" > /tmp/inventory; ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook -i /tmp/inventory ${path.module}/ansible/play.yml"
+    command = <<EOF
+      echo "[drone]\n${aws_instance.drone.public_ip}" > /tmp/inventory;
```

Comment:
let's chat about when you need `private_ip`, and also a different ssh key for ansible